runtime/packages/core-modules/assembly/logging.ts

```diff
@@ -0,0 +1,15 @@
+import { Serialiser } from '@as-substrate/core-utils';
+import { ScaleString } from 'as-scale-codec';
+import { ext_misc_print_utf8_version_1 } from './env';
+
+export class Logging {
```

Comment:
If you think about how this module will be used, I think it would be better to rename it to: `Log`
So that the dev experience is the following:
`Log.printUtf8()`